tests/src/common/WskTestHelpers.scala

```diff
@@ -118,3 +118,22 @@ trait WskTestHelpers extends Matchers {
         }
     }
 }
+
+object JsObjectExtensions {
```

Comment:
I'd call it `JsHelpers` and make it a `trait` we can mix in, but that's just personal preference.